src/adcm_pytest_plugin/plugin.py

```diff
@@ -40,6 +44,13 @@ def some_func():
     """
     global options  # pylint: disable=global-statement,invalid-name,global-variable-not-assigned
     options.__dict__.update(config.option.__dict__)
+    if config.option.actions_report_dir:
+        pytest.action_run_storage = []
+        pytest.actions_spec_storage = {}
+        try:
+            shutil.rmtree(_get_actions_dir(config))
+        except FileNotFoundError:
+            pass
```

Comment:
If we run tests with `xdist` this can break our report

Reply:
How?

Reply:
Each test will run `rmtree`

Reply:
Hook will be called only once, as I understand.

Reply:
The hook will be called ones per thread

Reply:
Hook will be called for each thread, but this logic actually does not break the report